Documentation/ceph-cluster-crd.md

```diff
@@ -287,6 +287,7 @@ You can set resource requests/limits for Rook components through the [Resource R
 - `mon`: Set resource requests/limits for mons
 - `osd`: Set resource requests/limits for OSDs
 - `rbdmirror`: Set resource requests/limits for RBD Mirrors
+- `prepareosd`: Set resource requests/limits for OSD prepare job
```

Comment:
Should we have a minimum value for the resources similar to other daemons? For example, 100MB

Reply:
I've thought about that, the problem is that today we have no idea of what's needed, 50MB might be bare minimum but that's only based on the little profiling from https://github.com/rook/rook/pull/4083#discussion_r334868467. That's why I decided to leave it up to the admin, at least until we get some numbers.

Reply:
Should we add a comment here about recommended value and that 50-100MB is expected to be sufficient? Otherwise the admin has no idea what to set.